src/command_modules/azure-cli-iot/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+0.3.5
++++++
+* Increment IotHub mgmt SDK requirement.
+* Minor fixes
+* Update test recordings.
```

Comment:
None of these entries should be in the history. The entry should detail the newly added parameters.